src/io/ResultWriter.cpp

```diff
@@ -1,89 +1,111 @@
 #include "io/ResultWriter.h"
 
 #include "Domain.h"
-#include "parallel/DomainDecompBase.h"
 #include "Simulation.h"
+#include "parallel/DomainDecompBase.h"
 #include "utils/Logger.h"
-#include <chrono>
+
+#include <fstream>
 
 using Log::global_log;
 
 void ResultWriter::readXML(XMLfileUnits& xmlconfig) {
-	_writeFrequency = 1;
 	xmlconfig.getNodeValue("writefrequency", _writeFrequency);
 	global_log->info() << "[ResultWriter] Write frequency: " << _writeFrequency << endl;
 
-	_outputPrefix = "mardyn";
 	xmlconfig.getNodeValue("outputprefix", _outputPrefix);
 	global_log->info() << "[ResultWriter] Output prefix: " << _outputPrefix << endl;
 
-	size_t acc_steps = 1000;
-	xmlconfig.getNodeValue("accumulation_steps", acc_steps);
-	_U_pot_acc = new Accumulator<double>(acc_steps);
-	_p_acc = new Accumulator<double>(acc_steps);
-	global_log->info() << "[ResultWriter] Accumulation steps: " << acc_steps << endl;
-
-	_writePrecision = 5;
 	xmlconfig.getNodeValue("writeprecision", _writePrecision);
 	global_log->info() << "[ResultWriter] Write precision: " << _writePrecision << endl;
 }
 
 void ResultWriter::init(ParticleContainer * /*particleContainer*/,
                         DomainDecompBase *domainDecomp, Domain * /*domain*/) {
 
+	// Only main rank writes data to file
 	if(domainDecomp->getRank() == 0) {
-		const auto now = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-		tm unused{};
-		const auto nowStr = std::put_time(localtime_r(&now, &unused), "%c");
-		string resultfile(_outputPrefix+".res");
-		_resultStream.open(resultfile.c_str(), std::ios::out);
-		_resultStream << "# ls1 MarDyn simulation started at " << nowStr << endl;
-		_resultStream << "# Averages are the accumulated values over " << _U_pot_acc->getWindowLength()  << " time steps."<< endl;
-		_resultStream << std::setw(10) << "# step" << std::setw(_writePrecision+15) << "time" 
-			<< std::setw(_writePrecision+15) << "U_pot"
+		const string resultfile(_outputPrefix+".res");
+		std::ofstream resultStream;
+		resultStream.open(resultfile.c_str(), std::ios::out);
+		resultStream << std::setw(10) << "#step"
+			<< std::setw(_writePrecision+15) << "time"
 			<< std::setw(_writePrecision+15) << "U_pot_avg"
-			<< std::setw(_writePrecision+15) << "p"
+			<< std::setw(_writePrecision+15) << "U_kin_avg"
+			<< std::setw(_writePrecision+15) << "U_kinTrans_avg"
+			<< std::setw(_writePrecision+15) << "U_kinRot_avg"
 			<< std::setw(_writePrecision+15) << "p_avg"
-			<< std::setw(_writePrecision+15) << "beta_trans"
-			<< std::setw(_writePrecision+15) << "beta_rot"
 			<< std::setw(_writePrecision+15) << "c_v"
 			<< std::setw(_writePrecision+15) << "N"
```

Comment:
Why not create a second member like `_writeWidth = _writePrecision + 15`? Makes changes easier.